exporter/stackdriver/trace.go

```diff
@@ -116,8 +116,12 @@ func (e *traceExporter) uploadToStackdriver(spans []*trace.SpanData) {
 	for _, span := range spans {
 		req.Spans = append(req.Spans, protoFromSpanData(span, e.projectID))
 	}
-	err := e.client.BatchWriteSpans(context.Background(), &req)
+	// create a never-sampled span to prevent traces associated with exporter
+	span := trace.NewSpan("ignored: uploadToStackdriver", nil, trace.StartOptions{Sampler: trace.NeverSample()})
+	ctx := trace.WithSpan(context.Background(), span) // TODO: add timeouts
```

Comment:
Add some useful debugging infos:
* number of exported spans. As an attribute probably.
* log the error as status.